### DISCUSSION TITLE
Change pivot point of an object

### DISCUSSION THREAD

**Discussion by jluism** [google/model-viewer#2184] [General]
Hi

I have an object you can see in [this web](https://www.asturnatura.com/modelos3d/dolmen-de-lapos-espanal/4.html) and as you can see it pivot on the center of the geometry but i would like that rotation happens around above stones, when you rotate and zoom they dissapear or are located up of the image. Could this be changed?

Thanks

**Comment by elalish** [comment]
I believe you're looking for the camera-target: https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-cameraTarget

**Reply by jluism** [reply]
Thanks! It worked with this modification.
